nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -140,6 +140,50 @@ let
         ssl_stapling_verify on;
       ''}
 
+      ${optionalString (cfg.recommendedBrotliSettings) ''
+        brotli on;
+        brotli_static on;
+        brotli_comp_level 5;
+        brotli_window 512k;
+        brotli_min_length 256;
+        brotli_types
+          application/atom+xml
+          application/javascript
+          application/json
+          application/ld+json
+          application/manifest+json
+          application/rss+xml
+          application/vnd.geo+json
+          application/vnd.ms-fontobject
+          application/x-font-opentype
+          application/x-font-truetype
+          application/x-font-ttf
+          application/x-javascript
+          application/x-web-app-manifest+json
+          application/xhtml+xml
+          application/xml
+          font/eot
+          font/opentype
+          font/otf
+          font/truetype
+          image/bmp
+          image/svg+xml
+          image/vnd.microsoft.icon
+          image/x-icon
+          image/x-win-bitmap
+          text/cache-manifest
+          text/css
+          text/javascript
+          text/plain
+          text/vcard
+          text/vnd.rim.location.xloc
+          text/vtt
+          text/x-component
+          text/x-cross-domain-policy
+          text/xml;
```

Comment:
I would recommend to dedupe this list with gzip and make them one.

Reply:
Gzip and brotli have different parameter names.

Reply:
I mean that we can move the mime types into a nix variable and re-use them for both compressions.

Reply:
Fixed. Removed duplicates mime types.
Updated PR.